spec/features/user_sign_up_spec.rb

```diff
@@ -0,0 +1,83 @@
+require 'spec_helper'
+
+feature "User Sign Up", js: true, vcr: true do
+  background do
+    visit register_path
+  end
+
+  scenario "with valid personal info and credit card" do
+    enter_valid_personal_info
+    submit_with_valid_card
+
+    expect(page).to have_content "You have successfully registered. You can sign in now!"
+  end
+
+  scenario "with valid personal info but invalid credit card" do
```

Comment:
Since the card is expired I would probably title the test that way instead of `invalid`.